dasharo-stability/capsule-update.robot

```diff
@@ -25,10 +25,14 @@ Suite Setup         Run Keywords
 ...                     AND
 ...                     Check If CAPSULE FW FILE Is Present
 ...                     AND
-...                     Flash Firmware If Not QEMU
+...                     Prepare For Logo Persistence Test
 ...                     AND
 ...                     Upload Required Files
```

Comment:
This should happen after firmware is flashed, in case previous firmware didn't boot, unless of course one of the earlier setup keywords does it.

Reply:
Thing is that we want to flash the FW with prepared logo to check if the capsule update changes it. So it needs to be added before flashing.

Reply:
@krystian-hebel, please take a look ^

Reply:
Yes, `Prepare For Logo Persistence Test` must happen before flashing, but `Upload Required Files` must take place after flashing.

Reply:
@krystian-hebel Changed the order in: a6d1871fd413293000eb2c8994413e33e55b3654
Actually it simplifies suite prepare as there is no need to boot to ubuntu twice to upload files and get system data.